Joinrpg/Helpers/StringFormatHelper.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using JetBrains.Annotations;
+
+namespace JoinRpg.Web.Helpers
+{
+  public static class StringFormatHelper
+  {
+    /// <summary>
+    /// Returns a substring with first occurrence of <param name="targetText"></param> in the middle
+    /// and not longer than the given maxLength
+    /// </summary>
+    public static string TruncateString(
```

Comment:
According to the other comments I will make it a private method of a VM then. So no unit test is needed here.